### DISCUSSION TITLE
Does longhorn can working well on CentOS 7.9?

### DISCUSSION THREAD

**Discussion by hunter-xue** [longhorn/longhorn#5690] [Q&A]
Runing longhorn on Ubuntu is well, but always got error below on CentOS 7:

Warning  FailedAttachVolume  60s (x124 over 4h22m)  attachdetach-controller  AttachVolume.Attach failed for volume "longhorn-test-vol" : rpc error: code = DeadlineExceeded desc = volume longhorn-test-vol failed to attach to node 10.100.129.195

**Comment by derekbit** [comment]
Can you check the environment by [environment_check.sh](https://longhorn.io/docs/1.4.1/deploy/install/#using-the-environment-check-script)? If all requirements are met, you can disable the SELinux temporary and see if it works.

**Comment by hunter-xue** [comment]
Run env check script is fine, and diabled SELinux also...

**Reply by derekbit** [reply]
Can you help check longhorn-manager's log?

**Reply by hunter-xue** [reply]
Found logs like this, but no more details in it...
![image](https://user-images.githubusercontent.com/4905312/229403533-00c9e069-5d8e-4541-bbee-b935728878f0.png)

**Comment by hunter-xue** [comment]
missed one step for CentOS/RHEL, working well now...
![image](https://user-images.githubusercontent.com/4905312/229758614-f94cfe91-b512-409b-9461-17b9554486b6.png)
